# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/airspy/hotfix-bool.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- airspyone_host-1.0.10/libairspy/src/airspy.c.vanilla	2026-09-05 00:02:46.217774785 +0300
+++ airspyone_host-1.0.10/libairspy/src/airspy.c	2026-09-05 00:02:56.215469016 +0300
@@ -35,9 +35,9 @@
 #include "filters.h"
 
 #ifndef bool
-typedef int bool;
-#define true 1
-#define false 0
+//typedef int bool;
+//#define true 1
+//#define false 0
 #endif
 
 #define PACKET_SIZE (12)
--- airspyone_host-1.0.10/airspy-tools/src/airspy_gpio.c.vanilla	2026-09-05 00:02:10.863877448 +0300
+++ airspyone_host-1.0.10/airspy-tools/src/airspy_gpio.c	2026-09-05 00:02:20.751982599 +0300
@@ -27,9 +27,9 @@
 #include <getopt.h>
 
 #ifndef bool
-typedef int bool;
-#define true 1
-#define false 0
+//typedef int bool;
+//#define true 1
+//#define false 0
 #endif
 
 #define PORT_NUM_INVALID (255)
--- airspyone_host-1.0.10/airspy-tools/src/airspy_gpiodir.c.vanilla	2026-09-05 00:01:52.007176879 +0300
+++ airspyone_host-1.0.10/airspy-tools/src/airspy_gpiodir.c	2026-09-05 00:02:01.088375184 +0300
@@ -27,9 +27,9 @@
 #include <getopt.h>
 
 #ifndef bool
-typedef int bool;
-#define true 1
-#define false 0
+//typedef int bool;
+//#define true 1
+//#define false 0
 #endif
 
 #define PORT_NUM_INVALID (255)
--- airspyone_host-1.0.10/airspy-tools/src/airspy_info.c.vanilla	2026-09-05 00:01:31.633642904 +0300
+++ airspyone_host-1.0.10/airspy-tools/src/airspy_info.c	2026-09-05 00:01:42.064282056 +0300
@@ -29,9 +29,9 @@
 #include <getopt.h>
 
 #ifndef bool
-typedef int bool;
-#define true 1
-#define false 0
+//typedef int bool;
+//#define true 1
+//#define false 0
 #endif
 
 #define AIRSPY_MAX_DEVICE (32)
--- airspyone_host-1.0.10/airspy-tools/src/airspy_r820t.c.vanilla	2026-09-05 00:01:09.256022461 +0300
+++ airspyone_host-1.0.10/airspy-tools/src/airspy_r820t.c	2026-09-05 00:01:19.829896349 +0300
@@ -27,9 +27,9 @@
 #include <getopt.h>
 
 #ifndef bool
-typedef int bool;
-#define true 1
-#define false 0
+//typedef int bool;
+//#define true 1
+//#define false 0
 #endif
 
 #define REGISTER_NUM_MIN (0)
--- airspyone_host-1.0.10/airspy-tools/src/airspy_rx.c.vanilla	2026-09-05 00:00:40.534497008 +0300
+++ airspyone_host-1.0.10/airspy-tools/src/airspy_rx.c	2026-09-05 00:00:51.015070089 +0300
@@ -37,9 +37,9 @@
 #define AIRSPY_RX_VERSION "1.0.5 23 April 2016"
 
 #ifndef bool
-typedef int bool;
-#define true 1
-#define false 0
+//typedef int bool;
+//#define true 1
+//#define false 0
 #endif
 
 #ifdef _WIN32
--- airspyone_host-1.0.10/airspy-tools/src/airspy_si5351c.c.vanilla	2026-09-05 00:00:15.775621002 +0300
+++ airspyone_host-1.0.10/airspy-tools/src/airspy_si5351c.c	2026-09-05 00:00:28.269991823 +0300
@@ -28,9 +28,9 @@
 #include <getopt.h>
 
 #ifndef bool
-typedef int bool;
-#define true 1
-#define false 0
+//typedef int bool;
+//#define true 1
+//#define false 0
 #endif
 
 static void usage() {
--- airspyone_host-1.0.10/airspy-tools/src/airspy_spiflash.c.vanilla	2026-09-04 23:59:35.019117250 +0300
+++ airspyone_host-1.0.10/airspy-tools/src/airspy_spiflash.c	2026-09-05 00:00:04.043639081 +0300
@@ -31,9 +31,9 @@
 #include <time.h>
 
 #ifndef bool
-typedef int bool;
-#define true 1
-#define false 0
+//typedef int bool;
+//#define true 1
+//#define false 0
 #endif
 
 #ifdef _WIN32

